build.sbt

```diff
@@ -17,10 +17,10 @@ version in ThisBuild := "0.9.0-SNAPSHOT"
 // Root project
 name := "root"
 
-version in ThisBuild := "0.9.0-SNAPSHOT"
-
 apiVersion in ThisBuild <<= version.map(extractApiVersion)
 
+scalaVersion in ThisBuild := "2.10.4"
```

Comment:
No, when i update that line to read 2.10.5 I still the the root project to mysteriously keep 2.10.4, and that leads to a bunch of warnings: 

```
[info] client/*:crossScalaVersions
[info]  List(2.10.5, 2.11.7)
[info] dsl/*:crossScalaVersions
[info]  List(2.10.5, 2.11.7)
[info] http4s-java-8/*:crossScalaVersions
[info]  List(2.10.4)
```

aha, I just figured it out!